ert_gui/plottery/plots/histogram.py

```diff
@@ -178,8 +179,8 @@ def _histogramLogBins(bin_count, minimum=None, maximum=None):
     @type data: pandas.DataFrame
     @rtype: int
     """
-    minimum = log10(float(minimum))
-    maximum = log10(float(maximum))
+    minimum = numpy.sign(float(minimum)) * log10(fabs(minimum))
```

Comment:
I think you mentioned the case of minimum being 0 when we talked, but how does this help against that? `fabs` of 0 is still 0 :)

Reply:
Well spotted. Do you think we should do something like this then?:
```python
if np.isclose(mimimum, 0):
   minimum = minumum + 0.0000001
```

Reply:
Is it possible to get number in the range 10E-12 - 10E-15?

In ert, if you really need them, you'll have to transform them after getting the range from e.g. 10E0-10E-3?